### DISCUSSION TITLE
How can I control the size of a boolean cell

### DISCUSSION THREAD

**Discussion by rakutek** [glideapps/glide-data-grid#564] [Q&A] [answered]
How can I control the size of a boolean cell' checkbox

When I increase rowHeight to make the image larger, the boolean cell appears larger. I would like to adjust this cell to appear smaller.
<img width="501" alt="スクリーンショット 2022-11-28 17 31 22" src="https://user-images.githubusercontent.com/42936741/204230298-9123508d-92f7-45b9-b97f-a1b46e529cbc.png">

**Comment by jassmith** [comment]
ooof, haha. Let me fix that...

**Comment by jassmith** [comment, marked as answer]
This is resolved in main and will be fixed in the upcoming point release.

**Reply by rakutek** [reply]
Thank you !!
